code/API_definitions/quality-on-demand.yaml

```diff
@@ -719,146 +630,6 @@ components:
-            Priority levels allow efficient resource allocation and ensure optimal performance
-            for various services in each technology, with the highest priority traffic receiving
-            preferential treatment.
-            The lower value the higher priority.
-            Not all access networks use the same priority range, so this priority will be
-            scaled to the access network's priority range.
-          format: int32
-          minimum: 1
-          maximum: 100
-        packetDelayBudget:
-          description: |
-            The packet delay budget is the maximum allowable one-way latency between the customer's device
-            and the gateway from the operator's network to other networks. By limiting the delay, the network
-            can provide an acceptable level of performance for various services, such as voice calls,
-            video streaming, and data.
-            The end-to-end or round trip latency will be about two times this value plus the latency not controlled
-            by the operator
-          allOf:
-            - $ref: "#/components/schemas/Duration"
-        jitter:
-          description: |
-            The jitter requirement aims to limit the maximum variation in round-trip
-            packet delay for the 99th percentile of traffic, following ITU Y.1540
-            standards. It considers only acknowledged packets in a session, which are
-            packets that receive a confirmation of receipt from the recipient (e.g.,
-            using TCP). This requirement helps maintain consistent latency, essential
-            for real-time applications such as VoIP, video calls, and gaming.
-          allOf:
-            - $ref: "#/components/schemas/Duration"
-        packetErrorLossRate:
-          type: integer
-          description: |
-            The exponential power of the allowable error loss rate 10^(-N).
-            For instance 3 would be an error loss rate of 10 to the power of -3 (0.001)
-
-            For 5G network the 3GPP specification TS 23.203 defines the packet error loss rate QCI attribute. It
-            describes the Quality of Service (QoS) Class Identifier (QCI) parameters used to
-            differentiate traffic classes in mobile networks, ensuring appropriate resource
-            allocation and performance for various services.
-
-            The packet error loss rate is one of the QCI attributes, providing information on the
-            acceptable packet loss rate for a specific traffic class. This attribute helps maintain
-            the desired performance level for services like voice calls, video streaming, or data
-            transfers within the 3GPP mobile network.
-          format: int32
-          minimum: 1
-          maximum: 10
-          example: 3
-      required:
-        - name
-        - status
-
     QosProfileName:
       description: |
         A unique name for identifying a specific QoS profile.
```

Comment:
I understand qos-profiles API is optional for quality-on-demand API. Would it be a good idea to refer to qos-profiles API? Something like `When qos-profile API is provided by the API provider, this QoSProfileName will be one from the profile set provided by the qos-profile API`

Or we could add this documentation to the description of this API in line 25, 45 or 51.

Reply:
Yes, I think it is convenient to document the relation with the other API, as now it is not self-contained in this API spec that other part.

Reply:
PR updated with a reference to the other API.